src/services/importTracker.ts

```diff
@@ -473,7 +475,10 @@ export function findModuleReferences(program: Program, sourceFiles: readonly Sou
 
         forEachImport(referencingFile, (_importDecl, moduleSpecifier) => {
             const moduleSymbol = checker.getSymbolAtLocation(moduleSpecifier);
-            if (moduleSymbol === searchModuleSymbol) {
+            const compilerOptions = program.getCompilerOptions();
+            const jsxImport = getJSXRuntimeImport(getJSXImplicitImportBase(compilerOptions, referencingFile), compilerOptions);

```

Comment:
I think jsxImport can be lifted out of the the loop.

Reply:
Fixed.